data/dpkg_templates/bulk-extractor-rules

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+
+# output every command that modifies files on the build system.
+#export DH_VERBOSE = 1
+
+%:
+	dh $@ --with autoreconf
+
+.PHONY: override_dh_auto_test
```

Comment:
Added this to prevent generation of test files that are not in the source package